From: Punit Agrawal <punit@debian.org>
Date: Sat, 20 Jul 2019 07:23:47 +0900
Subject: [PATCH] Respect PREFIX in GNUmakefile

---
 GNUmakefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/GNUmakefile b/GNUmakefile
index 49634d6..1d2b576 100644
--- a/GNUmakefile
+++ b/GNUmakefile
@@ -3,9 +3,9 @@
 ############################################################################
 
 # Set this to change the target installation path
-DESTDIR ?= /usr/local
-BINDIR   = ${DESTDIR}/bin
-MANDIR   = ${DESTDIR}/share/man
+PREFIX ?= /usr
+BINDIR   = ${DESTDIR}${PREFIX}/bin
+MANDIR   = ${DESTDIR}${PREFIX}/share/man
 DISTDIR ?= dist
 
 # filename of the executable
