From: Punit Agrawal <punitagrawal@gmail.com>
Date: Tue, 8 Apr 2014 18:43:21 +0100
Subject: Respect prefix in addition to DESTDIR in Makefile

---
 Makefile | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/Makefile b/Makefile
index f8f4039..b19194b 100644
--- a/Makefile
+++ b/Makefile
@@ -1,4 +1,4 @@
-DESTDIR ?= /usr/local
+PREFIX ?= /usr
 
 MAN_MD = $(wildcard man/*.md)
 ROFFS = $(MAN_MD:.md=)
@@ -34,8 +34,8 @@ release: build
 	git tag v`./direnv version`
 
 install: all
-	install -d $(DESTDIR)/bin
-	install -d $(DESTDIR)/share/man/man1
-	install direnv $(DESTDIR)/bin
-	cp -R man/*.1 $(DESTDIR)/share/man/man1
+	install -d $(DESTDIR)$(PREFIX)/bin
+	install -d $(DESTDIR)$(PREFIX)/share/man/man1
+	install direnv $(DESTDIR)$(PREFIX)/bin
+	cp -R man/*.1 $(DESTDIR)$(PREFIX)/share/man/man1
 
