From: Punit Agrawal <punitagrawal@gmail.com>
Date: Sun, 28 Jun 2015 13:30:11 +0100
Subject: Update Makefile to skip installing the manpages

---
 Makefile | 2 --
 1 file changed, 2 deletions(-)

diff --git a/Makefile b/Makefile
index 729e5ed..95303a4 100644
--- a/Makefile
+++ b/Makefile
@@ -37,9 +37,7 @@ test: build
 
 install: all
 	install -d $(DESTDIR)$(PREFIX)/bin
-	install -d $(DESTDIR)$(PREFIX)/share/man/man1
 	install direnv $(DESTDIR)$(PREFIX)/bin
-	cp -R man/*.1 $(DESTDIR)$(PREFIX)/share/man/man1
 
 dist:
 	go get github.com/mitchellh/gox
