From: Punit Agrawal <punit.agrawal@arm.com>
Date: Tue, 8 Apr 2014 19:09:56 +0100
Subject: Update Makefile to skip installing the manpages

---
 Makefile | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/Makefile b/Makefile
index 4e87fda..12f7123 100644
--- a/Makefile
+++ b/Makefile
@@ -32,7 +32,4 @@ release: build
 
 install: all
 	install -d bin $(DESTDIR)$(PREFIX)/bin
-	install -d man $(DESTDIR)$(PREFIX)/share/man/man1
 	install direnv $(DESTDIR)$(PREFIX)/bin
-	cp -R man/*.1 $(DESTDIR)$(PREFIX)/share/man/man1
-
