From: Punit Agrawal <punitagrawal@gmail.com>
Date: Sun, 28 Jun 2015 15:47:51 +0100
Subject: Update Makefile to use existing manpages

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 8dd5398..6950d8f 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ ROFFS = $(MAN_MD:.md=)
 
 .PHONY: all man html test install dist
 #all: build man test
-all: build man
+all: build
 
 build: direnv
 
