From: Punit Agrawal <punit@debian.org>
Date: Thu, 18 Jul 2019 21:18:16 +0900
Subject: Re-write grep pattern to avoid shell escape

---
 stdlib.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/stdlib.sh b/stdlib.sh
index 0eea7cb..6ed33c3 100755
--- a/stdlib.sh
+++ b/stdlib.sh
@@ -532,7 +532,7 @@ layout_anaconda() {
     conda=$(command -v conda)
   fi
   PATH_add "$(dirname "$conda")"
-  env_loc=$("$conda" env list | grep -- "^$env_name\s")
+  env_loc=$("$conda" env list | grep -- '^'"$env_name"'\s')
   if [[ ! "$env_loc" == $env_name*$env_name ]]; then
     if [[ -e environment.yml ]]; then
       log_status "creating conda environment"
