From: Punit Agrawal <punitagrawal@gmail.com>
Date: Sun, 28 Jun 2015 15:47:51 +0100
Subject: Update Makefile to use existing manpages

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 95303a4..2d9f979 100644
--- a/Makefile
+++ b/Makefile
@@ -11,7 +11,7 @@ endif
 
 .PHONY: all man html test install dist
 #all: build man test
-all: build man
+all: build
 
 build: direnv
 
