From: zimbatm <zimbatm@zimbatm.com>
Date: Mon Jul 13 20:10:01 2015 +0100
Origin: upstream, https://github.com/direnv/direnv/commit/9b60b43216d6c1c58ea1b9e74a483619acefbb65
Subject: Let bash inherit the PATH in the dump test

---
 test/scenarios/dump/.envrc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/scenarios/dump/.envrc b/test/scenarios/dump/.envrc
index 648c84a..4a73d18 100644
--- a/test/scenarios/dump/.envrc
+++ b/test/scenarios/dump/.envrc
@@ -2,5 +2,5 @@ direnv_load env \
 	LS_COLORS='*.ogg=38;5;45:*.wav=38;5;45' \
 	LESSOPEN='||/usr/bin/lesspipe.sh %s' \
 	THREE_BACKSLASHES='\\\' \
-	bash -l -c "direnv dump"
+	bash -c "direnv dump"
 
